tests/std/expected_results.txt

```diff
@@ -1,46 +1,2 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-
-# *** INSUFFICIENTLY NEW PUBLIC TOOLSET ***
-# TRANSITION, VS 2019 16.6p2

```

Comment:
I believe you should update the README to mention 16.6 Preview 2 in both locations (currently it mentions Preview 1, so if someone follows that and then runs `tests/std`, these `expected_results` will be incorrect for them).

Reply:
Heh, the downside of our script not mentioning this is I didn't even know it happened. Will do.

Reply:
oh, sorry, just did it real quick, wanted to get the green checkmark asap.

Reply:
Nothing to be sorry about! Getting to work together like this is a great thing about Git